### PR TITLE
Update OOTB Graphs for Resource and Service Pages

### PR DESCRIPTION
**Pull request by brett0000FF**
### What does this PR do? What is the motivation?

- Revise confusing and outdated structure for OOTB graphs on Service and Resource pages.
- Clarify that Avg Time per Request graph uses execution time.
- Use shortcode for graph overviews since the graphs are the same between each page.

### Previews

- [Service page](https://docs-staging.datadoghq.com/brett0000FF/DOCS-6751/tracing/services/service_page/#out-of-the-box-graphs)
- [Resource page](https://docs-staging.datadoghq.com/brett0000FF/DOCS-6751/tracing/services/resource_page/#out-of-the-box-graphs)

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Comment by brett0000FF** [comment]
@lee-brian : It appears that the set of graphs is the same between the Service and Resource page. Is that correct? I'm reusing the new content for both pages in this PR. Thanks!

**Comment by brett0000FF** [comment]
Offline feedback from Brian:
> **Average Time per Request** graph is built on sampled data (traces). Given the rest of the top graphs are from unsampled sources, we should mention this information in the docs.

File: content/en/tracing/services/resource_page.md [hunk 1]
```diff
@@ -65,11 +52,11 @@ Use the top right percentile selectors to zoom into a given percentile, or hover
 
 ## Dependency Map with Navigator
 
-You can also view a map of all of a resource's upstream and downstream service dependencies. With the Dependency Map Navigator, you can see the flow of services, with spans that go through a specific resource ( endpoint, database query, etc.) end-to-end, along with their request counts. 
+You can also view a map of all of a resource's upstream and downstream service dependencies. With the Dependency Map Navigator, you can see the flow of services, with spans that go through a specific resource ( endpoint, database query, etc.) end-to-end, along with their request counts.
```
Comment (cswatt) [inline content/en/tracing/services/resource_page.md:55]:
```suggestion
You can also view a map of all of a resource's upstream and downstream service dependencies. With the Dependency Map Navigator, you can see the flow of services, with spans that go through a specific resource (endpoint, database query, etc.) end-to-end, along with their request counts.
```

**Review by cswatt — APPROVED** [review]
very solid, everything lgtm except for one extraneous space!

**Pull request merged**